tap_slack/transform.py

```diff
@@ -34,8 +35,10 @@ def transform_json(stream, data, date_fields, channel_id=None):
                 timestamp = record.get(date_field, None)
                 if timestamp and isinstance(timestamp, str):
                     if stream == 'messages' or stream == "threads" and date_field == 'ts':
-                        record['thread_ts'] = timestamp
                         record[date_field] = decimal_timestamp_to_utc_timestamp(timestamp)
+                        if thread_ts is None:
```

Comment:
I don't think this is functioning as intended. The intention is to not overwrite `thread_ts` if the stream is `threads`. We may need to split the if statement into two parts like so: 

```
if stream == messages: 
  record['thread_ts'] = decimal_blabla(timestamp)
  record[date_field] = decimal_blablabla(timestamp)
elif stream == 'threads' and date_field == 'ts':
  if 'thread_ts' in record:
    record['thread_ts'] = decimal_timestamp...(record.get['thread_ts'])
    record[date_field]=  = decimal_blabla(timestamp)
else: 
  ...
```

there's probably a better way to write this but you get the idea

Reply:
Yep, I've made an update, but we couldn't call `decimal_timestamp_to_utc_timestamp(timestamp)` for `thread_ts` in transform module. If we do, then stream won't be found and we'll receive an error.
That's why I updated `streams.py` file to transform `tread_ts` value after actual call to slack API.